service/vlogs/server/server.go

```diff
@@ -41,24 +42,91 @@ func (vl *VLogsServer) ServeHTTP(rw http.ResponseWriter, req *http.Request) {
 		}
 		return
 	}
+	if req.URL.Path == "/queryPodList" {
+		err := vl.queryPodList(rw, req)
+		if err != nil {
+			http.Error(rw, fmt.Sprintf("query pod list error: %s", err), http.StatusInternalServerError)
+			log.Printf("query pod list error: %s", err)
+		}
+		return
+	}
```

Comment:
Duplicate code, consider reusing the same function implementation:
queryConvert func( name string) return query(rw, req)

if err := queryConvert(req.URL.Path); err !=nil { return xxx}

For error output, please introduce other log component libraries, similar to the output of `logger.Error()`